Lib/platform.py

```diff
@@ -199,6 +200,10 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
         binary = f.read(chunksize)
         pos = 0
         while pos < len(binary):
+            if b'musl' in binary:
```

Comment:
Please move this case into the if-else right below as elif after the more common libc / GLIBC case.

Reply:
The motivation to put it before was to check it as soon as possible to break the loop, but also because the `libc` bytes will be found in a  musl binary, so having:
```
if b'libc' in binary or b'GLIBC' in binary:
    ...
elif b'musl' in binary:
    ...
```
first will be true, and m will have a no match regular expression.

I think we either put first the musl condition, or keep it as it's at the moment.

What do you think @malemburg  ?

Reply:
Fair enough. Please add a comment explaining this, though.